app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -865,6 +865,11 @@ def build_toolbar_hide_button(id)
       end
     when "OrchestrationTemplate", "OrchestrationTemplateCfn", "OrchestrationTemplateHot", "OrchestrationTemplateAzure"
       return true unless role_allows(:feature => id)
+    when "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfiguredSystem", "ManageIQ::Providers::Foreman::ConfigurationManager::ConfiguredSystem"
+      case id
```

Comment:
Should this be `if id == "configured_system_provision"` since there are no other cases?